released/packages/coq-simple-io/coq-simple-io.1.1.0/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+maintainer: "Li-yao Xia <lysxia@gmail.com>"
+authors: "Li-yao Xia"
+homepage: "https://github.com/Lysxia/coq-simple-io"
+bug-reports: "https://github.com/Lysxia/coq-simple-io/issues"
+license: "MIT"
+dev-repo: "git+https://github.com/Lysxia/coq-simple-io.git"
+build: [
+  [make "build"]
+  [make "test"] {with-test}
+]
+install: [make "install"]
+depends: [
+  "ocaml"
+  "coq" {>= "8.8" & "8.10~"}
```

Comment:
Everything looks, good now, but please change this for precision (you can use my suggestion to merge directly on GitHub):
```suggestion
  "coq" {>= "8.8" & < "8.10~"}
```

Reply:
Great, thanks!